wormhole-connect/src/hooks/useFetchRedeemTx.ts

```diff
@@ -0,0 +1,67 @@
+import { useContext, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { setRedeemTx } from 'store/redeem';
+import { RouteContext } from 'contexts/RouteContext';
+import { sleep } from 'utils';
+import {
+  api,
+  toChain,
+  TransferState,
+  UniversalAddress,
+} from '@wormhole-foundation/sdk';
+import config from 'config';
+
+// TODO: this hook is a stop-gap until the SDK reliably
+// sets the redeem tx hash on the receipt
+const useFetchRedeemTx = (): void => {
+  const dispatch = useDispatch();
+  const { receipt } = useContext(RouteContext);
+
+  useEffect(() => {
+    if (
+      !receipt ||
+      receipt.state < TransferState.Attested ||
+      'originTxs' in receipt === false ||
+      receipt.originTxs.length === 0
+    ) {
+      return;
+    }
+
+    let isActive = true;
+
+    const fetchRedeemTx = async () => {
+      const wormholeApi = config.wormholeApi.replace(/\/$/, '');
```

Comment:
TODO, make `api` package of new SDK handle this internally :P

Reply:
good idea: https://github.com/wormhole-foundation/wormhole-sdk-ts/pull/661